src/WireFormat.h

```diff
@@ -1605,6 +1648,182 @@ struct TakeIndexletOwnership {
     } __attribute__((packed));
 };
 
+struct TxParticipant {
```

Comment:
Need overall documentation for the structure, and for TxDecision below. The only things that don't need overall documentation are those for RPCs, since they all have the same general format, and there is also a method for each RPC, which documents it.